lib/hooks/core/hook-http.js

```diff
@@ -101,8 +101,6 @@ function uriFromOptions(options) {
   var uri;
   if (typeof options === 'string') {
     uri = options;
-  } else if (options.uri && options.uri.href) {
```

Comment:
I can't find any mention of a uri option in the http spec: 

https://nodejs.org/api/http.html#http_http_request_options_callback